src/app/static/src/app/components/plots/download/downloadButton.vue

```diff
@@ -0,0 +1,40 @@
+<template>
```

Comment:
Oh yeah PascalCase of course, it was unintentional. Since we will implement download across the charts, I think it's safe to have a generic download button as well.